appveyor.yml

```diff
@@ -75,7 +75,7 @@ test_script:
       & coverage erase
 
       $clcachePy = "$((Get-Location).Path)\clcache.py"
-      $env:CLCACHE_CMD = "coverage run -a --branch --include=$clcachePy $clcachePy";
+      $env:CLCACHE_CMD = "coverage run --parallel-mode --branch --include=$clcachePy $clcachePy";
```

Comment:
This part I don't understand - without `-a`, won't the integration test run overwrite the coverage data accumulated during the unit test run? And why is `--parallel-mode` needed?